cloud_recording.go

```diff
@@ -0,0 +1,81 @@
+package zoom // Use this file for /recording endpoints
+
+import "fmt"
+
+const (
+	// ListAllRecordingsPath - v2 lists all recordings
+	ListAllRecordingsPath = "/users/%s/recordings"
+)
+
+// RecordingFile represents a recordings file object
+type RecordingFile struct {
+	ID             string `json:"id"`
+	MeetingID      string `json:"meeting_id"`
+	RecordingStart string `json:"recording_start"`
+	RecordingEnd   string `json:"recording_end"`
+	FileType       string `json:"file_type"`
+	FileSize       int64  `json:"file_size"`
+	PlayURL        string `json:"play_url"`
+	DownloadURL    string `json:"download_url"`
+	Status         string `json:"status"`
+	DeletedTime    string `json:"deleted_time"`
+	RecordingType  string `json:"recording_type"`
+}
+
+// Meeting represents a zoom meeting object
+type Meeting struct {
+	UUID           string          `json:"uuid"`
+	ID             int             `json:"id"`
+	AccountID      string          `json:"account_id"`
+	HostID         string          `json:"host_id"`
+	Topic          string          `json:"topic"`
+	StartTime      *Time           `json:"start_time"`
+	Timezome       string          `json:"timezone"`
+	Duration       int             `json:"duration"`
+	TotalSize      int             `json:"total_size"`
+	RecordingCount int             `json:"recording_count"`
+	Type           int             `json:"type"`
```

Comment:
I was wrong on this—since these meeting objects are not the same as the ones from the meetings endpoint, there is no `Type` field. Please remove this field.